packages/codegen-ui-react/lib/utils/forms/validation.ts

```diff
@@ -119,3 +120,1245 @@ export const validateField = (value: any, validations: FieldValidationConfigurat
   }
   return { hasError: false };
 };
+
```

Comment:
I kept non generated validation function for unit tests. The tests can exist in the test package to test generated validation function as well